src/components/KSelect/KSelectItems.vue

```diff
@@ -70,6 +70,9 @@ const getGroupingItems = (grouping: string) => props.items.filter(item => item.g
   font-weight: 600;
   margin-bottom: var(--spacing-xxs);
   padding: var(--spacing-xs);
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
```

Comment:
You shouldn't need the vendor prefixes: https://caniuse.com/?search=user-select

Reply:
Removed